index.html

```diff
@@ -8,47 +8,324 @@
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <title>Desktop version</title>
     <link rel="stylesheet" href="style.css" />
-    <title>Setup and mobile version</title>
   </head>
   <body>
-    <header>
-      <nav class="nav">
-        <h2 class="name">Mucyo</h2>
-
-        <i class="fa fa-bars"></i>
+    <div class="container">
+      <nav>
+        <div class="logo">LOGO</div>
+        <div class="menu">
+          <ul>
+            <li><a href="#home">Home</a></li>
+            <li><a href="#news">News</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li>
+              <a href="#about">
+                <i class="fas fa-envelope"></i>
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="humberger">
+          <i class="fa fa-bars"></i>
+        </div>
       </nav>
-    </header>
-    <section class="headline-page">
-      <div class="introduction">
-        <h1 class="intro">Hey There.<br />I'm Mucyo</h1>
-        <h2 class="headline-page-h2">I'am a Software Developer</h2>
-
-        <p class="text-1">
-          I can help you build a product, feature or website <br />Look through
-          some of my work and experience!<br />
-          if you like what you see and have a project <br />
-          you need coded, don't hestiate to contacta me.
-        </p>
```

Comment:
[ OPTIONAL ] Please follow the font color and font family defined in the design.